test/AutoDiff/forward_mode_sil.swift

```diff
@@ -0,0 +1,99 @@
+// RUN: %target-swift-frontend -emit-sil -Xllvm -run-jvp-generation  -verify %s | %FileCheck %s -check-prefix=CHECK-DATA-STRUCTURES
```

Comment:
```suggestion
// RUN: %target-swift-frontend -emit-sil -Xllvm -run-jvp-generation -verify %s | %FileCheck %s -check-prefix=CHECK-DATA-STRUCTURES
```

Reply:
Wanted to double check whether we want a simpler flag here as well or not. I'm guessing no since the only difference is the addition of `-Xllvm -run-jvp-generation` for each line when compared to existing tests like `control_flow_sil.swift`

Reply:
No need for extra command, avoid extra command when possible.
If just passing the flags is possible, do that please.